rpms/SPECS/wazuh-manager.spec

```diff
@@ -298,6 +299,14 @@ if [ $1 = 2 ]; then
   fi
 fi
 
+VD_FILENAME='vd_1.0.0_vd_4.8.0.tar.xz'
+if [ -f "%{_localstatedir}/${VD_FILENAME}" ]; then
+    tar -xf %{_localstatedir}/${VD_FILENAME} -C %{_localstatedir}
+    chown wazuh:wazuh %{_localstatedir}/queue/vd
+    chown wazuh:wazuh %{_localstatedir}/queue/vd-updater
```

Comment:
This folder isn't required.

Reply:
As per the requirements, the vd_updater folder needs to be compressed.

Reply:
Ok, thank you.